### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by adelajaOlamilekan**
​
## Description
This PR allows users that were created by the start recording endpoint to be able to log in later.

​
## Related Issue (Link to linear ticket)

​
## Motivation and Context
This fix is needed because these users need to be able to login later to view all their videos

​
## How Has This Been Tested?

​
## Screenshots (if appropriate - Postman, etc):
Screenshot (1271)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/services/services.py [hunk 1]
```diff
@@ -415,3 +416,15 @@ def is_logged_in(request: Request) -> bool:
...
+
+    pw_bytes = password.encode("utf-8")
+
+    # generating the salt
+    salt = bcrypt.gensalt()
+
+    # Hashing the password
+    hashed_password = bcrypt.hashpw(pw_bytes, salt)
+
+    return hashed_password
```
Comment (zxenonx) [inline app/services/services.py:430]:
```suggestion
    return  bcrypt.hashpw(pw_bytes, salt)
```

File: tests/test_blob.py [hunk 2]
```diff
@@ -3,24 +3,27 @@
...
+# if sys.argv[1] == "--local":
+#     URL = LOCAL_URL
+# elif sys.argv[1] == "--remote":
+#     URL = REMOTE_URL
+# else:
+#     print("Invalid argument. Use '--local' or '--remote'")
+#     sys.exit()
+
+# GET_VIDEO_ID_URL = f"{URL}/start-recording/"
+# ENDPOINT_URL = f"{URL}/upload-blob/"
```
Comment (zxenonx) [inline tests/test_blob.py:24]: Can we remove these commented lines?

**Review by zxenonx — CHANGES_REQUESTED** [review]
Thanks @adelajaOlamilekan 
Kindly check the comments.

**Review by zxenonx — APPROVED** [review]
Thanks

**Review by Cofucan — APPROVED** [review]
Looks good

**Pull request merged**